### ISSUE TITLE
Add ability to change group currency

### ISSUE THREAD

**Issue by taoeffect** [okTurtles/group-income#886]
### Problem

Currently the "change mincome" proposal only lets you change the mincome amount, not the currency.

### Solution

Make it possible to propose a change to the currency. @mmbotelho does this need designs?

**Comment by mmbotelho** [comment]
Yes it does @taoeffect. See issue #784. That needs to be sorted out before this is implemented.

**Comment by taoeffect** [comment]
@mmbotelho replied! https://github.com/okTurtles/group-income-simple/issues/784#issuecomment-612123241

**Comment by sandrina-p** [comment]
Whatever the solution we find at #784, probably it will answer this issue too because they are directly related. For that reason, I'm closing this issue.

Feel free to re-open it if you think it's worth having 2 similar/duplicated issues.

**Comment by taoeffect** [comment]
@sandrina-p Issue #784 seems to be the design side of this issue, so I'm re-opening this as the implementation side.
